python/boat_controller_node.py

```diff
@@ -109,6 +110,15 @@ def minVoltageCallBack(min_voltage_msg_instance):
     lock.release()
 
 
+def lowWindCallBack(low_wind_msg_instance):
+    lock.acquire()
+
+    global lowWind
+    lowWind = low_wind_msg_instance
+
+    lock.release()
```

Comment:
Should `minVoltageCallback()` and `lowWindCallback()` call `publishRudderWinchAngle()` before `lock.release()`?

Reply:
Good point. That is a bug that I missed. Thanks for pointing that out.